docs/content/commands/rclone_mount.md

```diff
@@ -90,7 +90,7 @@ systems are a long way from 100% reliable. The rclone sync/copy
 commands cope with this with lots of retries.  However rclone mount
 can't use retries in the same way without making local copies of the
 uploads. Look at the [file caching](#file-caching)
-for solutions to make mount mount more reliable.
```

Comment:
This needs to be changed in `cmd/mountlib/mount.go`  - `rclone_mount.md` is auto generated from there.